include/coreinit/performancemonitor.h

```diff
@@ -0,0 +1,206 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup coreinit_performancemonitor Performance Monitor
+ * \ingroup coreinit
+ *
+ * @{
+ */
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+/// Used to tell OSSetPerformanceMonitor() which arguments are valid.
+typedef enum PerfMonArg {
```

Comment:
To be consistent with the other coreinit enums this should be called something like `OSPerfMonArg` or `OSPerformanceMonitorArg`. Same goes for the other enums.